client/my-sites/plans/current-plan/current-plan-thank-you-card/paid-plan-thank-you-card.js

```diff
@@ -26,11 +26,36 @@ const INSTALL_STATE_COMPLETE = 1;
 const INSTALL_STATE_INCOMPLETE = 2;
 
 export class PaidPlanThankYouCard extends Component {
-	componentDidUpdate( prevProps ) {
-		if ( prevProps.progressComplete < 100 && this.props.progressComplete >= 100 ) {
-			this.props.recordTracksEvent( 'calypso_plans_autoconfig_success', {
+	tracksEventSent = false;
+
+	recordAutoconfigTracksEvent( eventName, options = {} ) {
```

Comment:
Might be nice to name this `recordAutoconfigTracksEventOnce` or add a comment to make it clear this will send a tracks event no more than 1 time.